<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_LineAngleRelOptions</class>
 <widget class="QWidget" name="LC_LineAngleRelOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="2">
    <widget class="QCheckBox" name="cbFree">
     <property name="toolTip">
      <string>If checked, the length of line is defined by mouse position instead of setting</string>
     </property>
     <property name="text">
      <string>Free</string>
     </property>
    </widget>
   </item>
   <item row="0" column="16">
    <widget class="QLineEdit" name="leOffset">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Offset of tick snap point from intersection point</string>
     </property>
    </widget>
   </item>
   <item row="0" column="18">
    <widget class="QCheckBox" name="cbDivide">
     <property name="toolTip">
      <string>If checked, original entity will be divided by intersection point.</string>
     </property>
     <property name="text">
      <string>Divide</string>
     </property>
    </widget>
   </item>
   <item row="0" column="15">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Offset:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QLineEdit" name="leDistance">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Distance of intersection point from specified line snap point</string>
     </property>
    </widget>
   </item>
   <item row="0" column="14">
    <widget class="QComboBox" name="cbTickSnapMode">
     <property name="toolTip">
      <string>Defines which part of created line will be snapped to intersection point</string>
     </property>
     <item>
      <property name="text">
       <string>Start</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Middle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Tick Snap:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="leLineSnapMode">
     <property name="text">
      <string>Line Snap:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leLength">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Length of line</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QCheckBox" name="cbRelativeAngle">
     <property name="toolTip">
      <string>If checked, angle is relative to angle of selected entity</string>
     </property>
     <property name="text">
      <string>Rel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Length:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QComboBox" name="cbLineSnapMode">
     <property name="toolTip">
      <string>Snap point for position of intersection point between created line and source entity (within original line)</string>
     </property>
     <item>
      <property name="text">
       <string comment="linesnap">Free</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Start</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Middle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="lblAngle">
     <property name="text">
      <string>Angle:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="19">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="11">
    <widget class="QLabel" name="lblDistance">
     <property name="text">
      <string>Snap Distance:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="leAngle">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Angle of line</string>
     </property>
    </widget>
   </item>
   <item row="0" column="17">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
